src/GraphicsControls/Platform/iOS/NativeGraphicsControlView.cs

```
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Diagnostics;
+using CoreGraphics;
+using Foundation;
+using Microsoft.Maui.Graphics.Native;
+using UIKit;
+
+namespace Microsoft.Maui.Graphics.Controls
+{
+	public class NativeGraphicsControlView : NativeGraphicsView
+	{
+        IGraphicsControl? _graphicsControl;
+		bool _pressedContained;
+
+		public NativeGraphicsControlView()
+		{
+			BackgroundColor = UIColor.Clear;
+		}
+
+		public IGraphicsControl? GraphicsControl
+		{
+			get => _graphicsControl;
+			set => Drawable = _graphicsControl = value;
+		}
+			
+		public override void TouchesBegan(NSSet touches, UIEvent? evt)
+		{
+			try
+			{
+				if (!IsFirstResponder)
+					BecomeFirstResponder();
+
+				var viewPoints = this.GetPointsInView(evt);
+				GraphicsControl?.StartInteraction(viewPoints);
+				_pressedContained = true;
+			}
+			catch (Exception exc)
+			{
+				Debug.WriteLine("An unexpected error occured handling a touch event within the control.", exc);
```

Is swallowing the exception intentional?